### PR TITLE
[ReadmeGenerator] Find README in a case-insensitive manner

### LINKED ISSUE #407: Filenames are case sensitive
Maya [isn't getting the QI boost](https://cocoapods.org/pods/Moya/quality) for having a changelog, and it looks like it's because [this function](https://github.com/CocoaPods/cocoadocs.org/blob/a9ec353364fc91101ac606b6b13eb962a8d37bcb/classes/readme_generator.rb#L40) does a case-sensitive compare. Since [switching to less scary filenames](https://github.com/Moya/Moya/commit/8a2f3673ee4279f92aed94a4082fbfd9a31bf3f8#diff-9b8fab691c00b9e5380b19ce882f3271), Moya hasn't been getting the boost :fearful: I'm not even 100% sure it's finding the correct read or is using a generated one.

Anyway, wasn't sure at what level to implement a fix, otherwise I'd have sent a PR. If anyone has direction on this, happy to send a change in.

### PR DESCRIPTION
**Pull request by segiddins**
Closes #407.

\c @ashfurrow

### CODE REVIEW

**Comment by ashfurrow** [comment]
Neeeaaaattttttttt

**Comment by orta** [comment]
wow, impressive work @segiddins

File: classes/readme_generator.rb [hunk 1]
```diff
@@ -39,12 +39,7 @@ def github_render(spec_readme_path, rendered_path)
 
   def file_local_path(name, spec)
     download_location = $active_folder + "/download/#{spec.name}/#{spec.version}/#{spec.name}/"
-    local_files = Dir.glob(download_location + "*")
-    first_folders = Dir.glob(download_location + "*/*")
-    files = local_files + first_folders
-    files.select do |f|
-      f.end_with?("#{name}.md") || f.end_with?("#{name}.markdown") || f.end_with?("#{name}.mdown") || f.end_with?(name)
-    end.first
+    Dir.glob("#{download_location}/{*,*/*}{.md,.markdown,.mdown,}", File::FNM_CASEFOLD).first
```
Comment (segiddins) [inline classes/readme_generator.rb]: This is wrong, it needs to include name in the glob pattern

**Comment by orta** [comment]
I'm on it :)

**Pull request merged**